config/config.toml.example

```diff
@@ -213,6 +213,10 @@ max-procs = 0
 # Memory size quota for tidb server, 0 means unlimited
 server-memory-quota = 0
 
+# Memory alert scale of memory size quota for tidb server, 0 means no alert
+# If the instance memUsage is more than `memory-alert` * `memory-quota`, it will records SQLs and mem-profile.
+server-memory-alert = 0.8
```

Comment:
how about reuse `server-memory-quota`?

Reply:
1. We need an option to control whether the alert feature is on/off.
2. Setting the ratio manually is meaning. It also can get which SQL uses too many memory but tidb not oom. (Maybe someone wants to set 0.5)